[shopsys] Unified using ENVs (#2040)
shopsys/shopsys: docs/installation/application-configuration.md

# Application Configuration

> The application is configurable by [symfony configuration files](https://symfony.com/doc/4.4/configuration.html#configuration-parameters) or via environment variables which allows you to overwrite them.

Reviewer: ```suggestion
The application is configurable by [Symfony configuration files](https://symfony.com/doc/4.4/configuration.html#configuration-parameters) or via environment variables which allows you to overwrite them.
```

!!! tip
    Host values can be modified or can be aliased for your Operating System via `/etc/hosts` or `C:\Windows\System32\drivers\etc\hosts`

## Configuration by environment variables

> Environment variables are really handy to configure the right setting in desired application environment.

Reviewer: ```suggestion
Environment variables are really handy to configure the right setting in the desired application environment.
```

Author: fixed

> You may want to set some setting in different way (such as production, test or CI servers).

Reviewer: ```suggestion
You may want to set some settings in a different way (such as production, test, or CI servers).
```
